test/io-tests/test_io.py

```diff
@@ -299,23 +334,6 @@ def test_expected_config_from_db_settings(defaultenv, role, expectedconfig):
     assert dumpconfig(configpath=config, env=env) == expected
 
 
-def test_read_db_setting(defaultenv):
-    """
-    Should be able to read db settings with current_setting.
-
-    See: https://github.com/PostgREST/postgrest/pull/1729#discussion_r572946461
-    """
-    env = {
-        **defaultenv,
-        "PGRST_DB_CONFIG": "true",
-    }
-    with run(env=env) as postgrest:
-        uri = "/rpc/get_guc_value?name=pgrst.jwt_secret"
-        response = postgrest.session.get(uri)
-
-        assert response.text == '"OVERRIDEREALLYREALLYREALLYREALLYVERYSAFE"'
```

Comment:
I don't think we need this test-case anymore, because now that PG 14 has arrived, and we are running tests on it, we can't create any of those "non-showable-gucs" anymore, that were a problem in the referenced discussion.